include/sdf/Element.hh

```diff
@@ -479,6 +491,28 @@ namespace sdf
     // The possible child elements
     public: ElementPtr_V elementDescriptions;
 
+    /// \brief The <include> element that was used to load this entity. For
+    /// example, given the following SDFormat:
+    /// <sdf version = '1.8'>
```

Comment:
nit Why the space between ` = `?

What does `xmllint` say about that?

Reply:
hmm, just a force of habit for other times I use `=` in code. `xmllint` does remove all spaces. Removed in https://github.com/osrf/sdformat/pull/509/commits/2631da14dff94ea7c66e2480be626ffd3e33e72e.